Src/WitsmlExplorer.Frontend/components/ContentViews/JobsView.tsx

```diff
@@ -146,13 +146,19 @@ export const JobsView = (): React.ReactElement => {
                 : jobInfo.status,
             cancel:
               jobInfo.isCancelable === true && jobInfo.status === "Started" ? (
-                <StyledButton
-                  key="downloadall"
-                  variant="outlined"
-                  onClick={() => cancelJob(jobInfo.id)}
-                >
-                  <Icon name="clear" />
-                </StyledButton>
+                <EdsProvider density="compact">
+                  <div style={{ display: "flex", justifyContent: "center" }}>
+                    <Button
+                      style={{ height: "22px", width: "22px" }}
```

Comment:
Add this styling to the button to avoid the huge hitbox:
```js
const StyledButton = styled(Button)`
  height: 22px;
  width: 22px;
  &::after {
    width: 22px;
    height: 22px;
  }
`;
```

Reply:
It might be useful to create a new component called `TableButton`  in the `StyledComponents` directory (not sure about the naming). This way, we can use the smaller-sized button in other tables as well.

Reply:
Done